lib/call.ts

```diff
@@ -26,6 +26,9 @@ class CallTranspiler extends base.TranspilerBase {
           this.emit('new');
         }
         var newExpr = <ts.NewExpression>node;
+        if (!newExpr.arguments) {
+          newExpr.arguments = <ts.NodeArray<ts.Expression>>([]);
```

Comment:
Modifying the TypeScript AST is verboten - TypeScript has internal, unexposed structures that depend on the AST not being modified. Changing properties on it has lead to very, very obscure bugs in the past.

Can you just handle this in `visitCall`?